tests/lib/pkgdb.sh

```diff
@@ -83,8 +83,8 @@ distro_name_package() {
 
 distro_install_local_package() {
     allow_downgrades=false
-    while [ -n "$1" ]; do
-        case "$1" in
+    while [ -n "${1:-}" ]; do
+        case "${1:-}" in
```

Comment:
with the while above this shouldn't be possible

```suggestion
        case "$1" in
```